test/functional/Java8andUp/src/org/openj9/test/vmArguments/VmArgumentTests.java

```diff
@@ -719,14 +719,11 @@ public void testEnvironmentVariableOrdering() {
 				argumentPositions.containsKey(openJ9JavaOptionsArg));
 		assertTrue(IBM_JAVA_OPTIONS+ SHOULD_COME_AFTER+OPENJ9_JAVA_OPTIONS, 
 				argumentPositions.get(ibmJavaOptionsArg).intValue() > argumentPositions.get(openJ9JavaOptionsArg).intValue());
+		assertTrue(OPENJ9_JAVA_OPTIONS + SHOULD_COME_AFTER + JAVA_TOOL_OPTIONS, 
+				argumentPositions.get(openJ9JavaOptionsArg).intValue() > argumentPositions.get(javaToolOptionsArg).intValue());
 		if (!isJava8 || !isIBM) {
-			assertTrue(OPENJ9_JAVA_OPTIONS + SHOULD_COME_AFTER + JAVA_OPTIONS, 
```

Comment:
This `if` would read better if inverted (`if (isJava8 && isIBM) { ...`).

Reply:
Do you mean `if (!(isJava8 && isIBM)) {` ?

Reply:
Updated.

Reply:
No, I meant 'invert' like you did with the `if` statement now on line 348.

Reply:
There is no else case for this one.

Reply:
Right - I guess I missed that you made a branch of that `if` statement unconditional.